content/css/concepts/colors/terms/color/color.md

```diff
@@ -1,38 +1,79 @@
 ---
-Title: 'color'
+Title: 'CSS Colors - color'
 Description: 'To set color property values for an element.'
 Subjects:
   - 'Web Development'
   - 'Web Design'
 Tags:
   - 'Style'
   - 'Colors'
+  - 'CSS'
+  - 'Design'
+
 CatalogContent:
   - 'learn-css'
   - 'paths/front-end-engineer-career-path'
   - 'paths/full-stack-engineer-career-path'
 ---
 
-To set color property values for an element.
+To set **color** property values for an element. <color> sets the color of the textual and decorative parts of an element and text.
+
 
 ## Syntax
 
 ```css
 color: <value>;
 ```
 
-The `<value>` can be the following:
+The `<value>` of the color property can be any of the following:
 
 - Name of the color (e.g., `aqua`, `khaki`, `red`)
-- RGB colors (e.g., `rgb(249, 2, 171)`)
-- Hexadecimal Colors (e.g., `#ff0000`)
+- RGB (red, green and blue) colors (e.g., `rgb(249, 2, 171)`)
+- Hexadecimal colors (e.g., `#ff0000`)
+
 
-## Example 1
+## Examples
+
+The `h1` tag is set to be orange in color and the `p` tag is set as pink in color: <!--The background color, black/#000000/rgb(0,0,0), is only set for context-->
```

Comment:
```suggestion
The `h1` tag is set to be orange in color and the `p` tag is set as pink in color: <!--The background color, black/#000000/rgb(0,0,0), is only set for context:
```

Reply:
*Basically what I'm asking is should make the blanked out comment visible to everyone?